meson.build

```diff
@@ -26,6 +26,9 @@ glib_dep = dependency('glib-2.0', version: '>=2.30.0')
 gio_unix_dep = dependency('gio-unix-2.0', version: '>=2.20')
 gee_dep = dependency('gee-0.8', version: '>=0.8.5')
 gexiv2_dep = dependency('gexiv2', version: '>=0.4.90')
+if gexiv2_dep.version().version_compare('>= 0.11')
+  add_global_arguments(['--define=NEW_GEXIV2_API'], language : 'vala')
```

Comment:
I'd prefer a `GEXIV2_0_11` define argument instead of this that is vague and not future-proof :slightly_smiling_face: